extensions/amp-story-auto-ads/0.1/test/test-amp-story-auto-ads.js

```diff
@@ -241,8 +242,9 @@ describes.realWin('amp-story-auto-ads', {
     });
   });
 
-  describe('creation of attributes', () => {
-    it('should not allow blacklisted attributes', () => {
+  // TODO(@calebcordry, #21618): Fails because ad type is undefined.
+  describe.skip('creation of attributes', () => {
+    it.skip('should not allow blacklisted attributes', () => {
```

Comment:
this is breaking because of https://github.com/ampproject/amphtml/pull/21618/files#diff-4a080b08c414c51e235cba638bff05baR442 is wrong.

Reply:
Fixed source code, and unskipped both tests in this file.